test/gpu/native/streamPrototype/genDriver.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# -----------------------------------------------------------------------------
+# Hack to work around cmake/ssl interference on test machine
+# -----------------------------------------------------------------------------
+
+SPACK_ROOT="/cray/css/users/chapelu/spack"
+SSL_HASH="leoarhg"
+function unload_ssl() {
+  tempMODPATH=$MODULEPATH
+  export MODULEPATH=''
+  eval `$SPACK_ROOT/bin/spack unload --sh /$SSL_HASH`
+  export MODULEPATH=$tempMODPATH
+}
+
+function load_ssl() {
+  tempMODPATH=$MODULEPATH
+  export MODULEPATH=''
+  eval `$SPACK_ROOT/bin/spack load --sh /$SSL_HASH`
+  export MODULEPATH=$tempMODPATH
+}
+
+# -----------------------------------------------------------------------------
+# General set up for GPU testing on test machine
+# -----------------------------------------------------------------------------
+
+# Clear any Chapel variables
+chpl_vars=$(env | grep -o "^CHPL_[^=]*")
+for var in $chpl_vars; do
+  unset "$var"
+done
+
+module purge
```

Comment:
Maybe I should just have this source `common-native-gpu.bash`. Though I kind of like having it inlined in the script just because it makes it easier to read and hack without having to jump around files.